metaci/release/migrations/0016_auto_20210511_2018.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.1.7 on 2021-05-11 20:18
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    def populate_version_number(apps, schema_editor):
+        Release = apps.get_model("release.Release")
+        Release.objects.filter(version_number__isnull=True).update(version_number="N/A")
+
+    def unpopulate_version_number(apps, schema_editor):
+        """Creating a NO-OP function for rollback migration capability."""
+        pass
+
+    dependencies = [
+        ("release", "0015_auto_20210407_0218"),
+    ]
+
+    operations = [
+        migrations.RunPython(populate_version_number, unpopulate_version_number),
```

Comment:
You can do this so that you don't need to define an empty unpopulate_version_number:

```suggestion
        migrations.RunPython(populate_version_number, migrations.RunPython.noop),
```